cli.py

```diff
@@ -73,10 +73,13 @@ class Converter(object):
     def __init__(self, bob_config):
         self.config = bob_config
 
-    def convert(self, path):
+    def convert(self, path, excluded_filenames=[]):
```

Comment:
Is the default argument intentionally a mutable? See http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments for an explication.